this does 20 iterations of 50ms (1 second)

which is too little time for the builders under load.

make it 10 seconds.
--
diff --git a/pueue/tests/helper/wait.rs b/pueue/tests/helper/wait.rs
index f2d4ca9..a5e678f 100644
--- a/pueue/tests/helper/wait.rs
+++ b/pueue/tests/helper/wait.rs
@@ -22,7 +22,7 @@ pub async fn wait_for_task_condition<F>(
 where
     F: Fn(&Task) -> bool,
 {
-    let tries = 20;
+    let tries = 200;
     let mut current_try = 0;
     while current_try <= tries {
         let state = get_state(shared).await?;
